### PR TITLE
Add android-studio-beta

### PR DESCRIPTION
**Pull request by tir38**
Second part of #8993

Current version / sha for beta: https://developer.android.com/studio/archive

Will rebase after #9071 is merged.

Additionally, if **adding a new cask**:

- [X] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
    _how can I test this before merging?_
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
    _how can I test this before merging?_
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked the cask was not [already refused].
- [X] Checked the cask is submitted to [the correct repo].

### CODE REVIEW

File: Casks/android-studio-beta.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+cask 'android-studio-beta' do
```
Comment (vitorgalvao) [inline Casks/android-studio-beta.rb:1]: `android-studio-preview-canary`

Comment (miccal) [reply Casks/android-studio-beta.rb:1]: I think @vitorgalvao means `android-studio-preview-beta`

Comment (vitorgalvao) [reply Casks/android-studio-beta.rb:1]: > I think @vitorgalvao means `android-studio-preview-beta`

I do, thank you. It was a copy and paste I forgot to edit. Fixed.

**Comment by vitorgalvao** [comment]
There’s an extraneous commit.

**Comment by tir38** [comment]
@victorpopkov yep just needed to rebase. Should be ready now.

**Pull request merged**